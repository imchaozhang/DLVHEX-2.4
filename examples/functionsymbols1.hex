q(z).
q(y).
p(f(f(X))) :- q(X).
r(X) :- p(X).
r(Y) :- r(f(Y)).
