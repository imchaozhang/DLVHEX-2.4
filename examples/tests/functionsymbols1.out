{q(z),q(y),r(f(f(z))),r(f(f(y))),r(f(y)),r(f(z)),r(z),r(y),p(f(f(y))),p(f(f(z)))}
